### PR TITLE
Added a normalizeLineName to DsbProvider to fix busnames

### PR DESCRIPTION
**Pull request by tgc-dk**
Also fixed a test in DsbProviderLiveTest.

### CODE REVIEW

**Comment by grote** [comment]
This looks good to me. @schildbach do you see issues?

File: enabler/src/de/schildbach/pte/DsbProvider.java [hunk 1]
```diff
@@ -99,4 +100,17 @@ protected Product normalizeType(final String type) {
 
         return super.normalizeType(type);
     }
+
+    // Busses line name is formattedas "42#Bus 42" but we just want "42"
```
Comment (schildbach) [inline enabler/src/de/schildbach/pte/DsbProvider.java]: Small typo here.

File: enabler/test/de/schildbach/pte/live/DsbProviderLiveTest.java [hunk 2]
```diff
@@ -76,7 +76,7 @@ public void suggestLocations() throws Exception {
     public void shortTrip() throws Exception {
         final QueryTripsResult result = queryTrips(
                 new Location(LocationType.STATION, "900000011", null, "Copenhagen Airport"), null,
-                new Location(LocationType.STATION, "551922500", null, "Billund Airport"), new Date(), true, Product.ALL,
+                new Location(LocationType.POI, "551922500", null, "Billund Airport"), new Date(), true, Product.ALL,
```
Comment (schildbach) [inline enabler/test/de/schildbach/pte/live/DsbProviderLiveTest.java]: Why this change? Is the 'Billund Airport' station not existing any more?

Comment (tgc-dk) [reply enabler/test/de/schildbach/pte/live/DsbProviderLiveTest.java]: The test failed for me without this change. There is no Billund Airport (train)station, but there exists bus stops.

File: enabler/src/de/schildbach/pte/DsbProvider.java (hunk 1 above)
Comment (tgc-dk) [reply enabler/src/de/schildbach/pte/DsbProvider.java]: Will fix

**Comment by tgc-dk** [comment]
Please re-review

**Comment by schildbach** [comment]
Merged. Thanks!
